src/commands/pull.ts

```diff
@@ -97,7 +103,7 @@ export const handler = async (args: Arguments<PullOptions>): Promise<unknown> =>
   try {
     const locales = args.locales?.split(',').filter(p => p) as Locale[] || []
     const provider = ProviderFactory(conf)
-    const messages = await provider.pull({ locales, dryRun, normalize, format })
+    const messages = await provider.pull({ locales, dryRun, normalize, format, providerArgs })
```

Comment:
I think `providerArgs` should be parsed to object, and it should be passed to provider.
Because it's user-friendly for providers.

Reply:
Sure!
Please wait a moment.